README.md

```diff
@@ -32,7 +32,7 @@ Prerequisites: C/C++ compiler, CMake (minimum version 3.10.2)
 ## <a name="dependencies"></a> Dependencies & external libraries
```

Comment:
Maybe add to this file, where to place pointcloud files, how to specify their coordinate systems etc.

Reply:
I think this is a good thing to have, but it could be done in another PR where we explain all directories (`osc`, `odr`, `pointclouds`, etc.) and what should be placed in them in order to get ATOS running as well as explaining `params.yaml`